composer.json

```diff
@@ -17,6 +17,7 @@
   },
   "require-dev": {
     "api-clients/test-utilities": "^1.0",
+    "container-interop/container-interop": "^1.1",
```

Comment:
Only required when using the ContainerLocator. Containers implementing this interface will always require it. So I think require-dev is sufficient.